### DISCUSSION TITLE
Show subtitles under subtitle offset slider

### DISCUSSION THREAD

**Discussion by Lenart12** [jellyfin/jellyfin-web#4364] [Ideas]
The largest painpoint for me when setting subtitle offsets is never being quite sure which way and for how much to move the slider. My idea that would simplify this is showing a moving subtitle track as shown in the picture below, that is moved left/right depending on current offset.

This would simplify the offset process like so.

1. User plays the video and notes what was last said.
2. He then pauses the video and adjusts the offset so that the correct subtitle box is over the middle line.

If the video is not paused subtitle boxes would move right to left according to the playback speed and changing the offset would move them more left/right. The scale of the timeline (e.g. -30s to +30s) could either be the same as the offset slider or hardcoded/user defined.

Here is an example sketchup:
![Web 1920 – 1](https://user-images.githubusercontent.com/39205323/219952009-4b7a31d9-1abb-4be3-bb3c-29d864a8d782.png)

And some alternative color schemes:
![image](https://user-images.githubusercontent.com/39205323/219952792-311f1a58-e342-4bda-9ae4-22d2e393157b.png)
![image](https://user-images.githubusercontent.com/39205323/219952797-c725353e-6f9d-4cb5-a081-c46ee050a4ba.png)
![image](https://user-images.githubusercontent.com/39205323/219952802-9bcb3abf-bbac-4c5d-b937-ae11a7325a3a.png)

**Comment by ThijmenDam** [comment]
I think replacing the ambiguous plus and minus signs with natural language would already suffice. For example, instead of "+1.5" you would see something like "1.5 seconds later".

**Comment by she11sh0cked** [comment]
Honestly, I think this is a really cool idea. It's also pretty unique insofar that i don't know of any service that implements subtitle syncing like this. If there isn't any obvious reason why no one has tried their hands at it yet, I might try implementing it. That being said, does anyone know why this isn't implemented anywhere?

**Comment by armanckeser** [comment]
https://github.com/jellyfin/jellyfin-web/pull/6735
